completion/runcode.cwl

```diff
@@ -53,48 +51,40 @@ inline
 \inln{program}{code%definition}
 \inln{program}{code%definition}[vbox|inline]
 
-\runJulia{source file}{output file}
-\runJulia{source file}{output file}[run|cache]
-\runJulia[server spec]{source file}{output file}
-\runJulia[server spec]{source file}{output file}[run|cache]
-\runJulia{file}{output file}#Si
-\runJulia[server spec]{file}{output file}#Si
+\runJulia{source file%file}{output file}
+\runJulia{source file%file}{output file}[run|cache]
+\runJulia[server spec]{source file%file}{output file}
+\runJulia[server spec]{source file%file}{output file}[run|cache]
 
 \inlnJulia{code%definition}
 \inlnJulia{code%definition}[vbox|inline]
 \inlnJulia[server spec]{code%definition}
 \inlnJulia[server spec]{code%definition}[vbox|inline]
 
-\runMatlab{source file}{output file}
-\runMatlab{source file}{output file}[run|cache]
-\runMatlab[server spec]{source file}{output file}
-\runMatlab[server spec]{source file}{output file}[run|cache]
-\runMatlab{file}{output file}#Si
-\runMatlab[server spec]{file}{output file}#Si
+\runMatlab{source file%file}{output file}
+\runMatlab{source file%file}{output file}[run|cache]
+\runMatlab[server spec]{source file%file}{output file}
+\runMatlab[server spec]{source file%file}{output file}[run|cache]
 
 \inlnMatlab{code%definition}
 \inlnMatlab{code%definition}[vbox|inline]
 \inlnMatlab[server spec]{code%definition}
 \inlnMatlab[server spec]{code%definition}[vbox|inline]
 
-\runR{source file}{output file}
-\runR{source file}{output file}[run|cache]
-\runR[server spec]{source file}{output file}
-\runR[server spec]{source file}{output file}[run|cache]
-\runR{file}{output file}#Si
-\runR[server spec]{file}{output file}#Si
+\runR{source file%file}{output file}

```

Comment:
shouldn't output file also be declared as file ? This is basically for the syntaxchecker to leave that argument alone

Reply:
Yes that's true. I'll add it to the next PR. This `%file` is a good addition since it allows things like being able to Ctrl+Click the arg in `\lstinputlisting{myfile.tex}` without adding the definitions, packages, etc. in myfile.tex to the completion in the original document.